packages/vuetify/src/components/VTooltip/VTooltip.ts

```diff
@@ -44,9 +44,6 @@ export default mixins(Colorable, Delayable, Dependent, Detachable, Menuable, Tog
       default: 'span',
     },
     transition: String,
-    zIndex: {
```

Comment:
Where did this go? Couldn't find this in any of the changed mixins

Reply:
Menuable